server/tilesource/base.py

```diff
@@ -287,58 +548,21 @@ def getRegion(self, width=None, height=None, **kwargs):
         :param height: maximum height in pixels.
         :param **kwargs: optional arguments.  Some options are encoding,
             jpegQuality, jpegSubsampling, top, left, right, bottom,
-            regionWidth, regionHeight, units ('pixels' or 'fraction').
+            regionWidth, regionHeight, units ('pixels' or 'fraction'),
+            magnification, mm_x, mm_y, exact.  See _tileIteratorInfo.
```

Comment:
Change `See _tileIteratorInfo` to `See tileIterator` since _tileIteratroInfo is a private function.

Also, i would suggest renaming `width, height` to something like `maxOutWidth, maxOutHeight` and `regionWidth,  regionHeight` to `width, height`. This would weed away most of the confusion.

Also, maybe make an alias function called `getROI` without the `maxOutWidth` and `maxOutHeight` parameters. This intention of this function should be solely to get an ROI at any magnification.

Also, i would suggest using `units=pixels` to mean specified resolution pixels and add `units=base_pixels` to mean maximum resolution pixels .